classes/class.usrtoHelper.php

```diff
@@ -157,12 +158,37 @@ protected function checkAccess($usr_id, $take_over_id, $group_id)
             return true;
         }
 
-        // If the user taking over is of id 13? or is not in the admin role he does not have permission.
-        if (!isset($usr_id) || $usr_id == 13 || !in_array(2, self::dic()->rbacreview()->assignedGlobalRoles($usr_id))) {
+        if(!$this->checkPluginAccess($usr_id)){
             ilUtil::sendFailure(self::plugin()->translate('no_permission'), true);
             ilUtil::redirect('login.php');
+        }
+    }
 
+    public  function checkPluginAccess($usr_id=null){
```

Comment:
add return value to method signature, probably: 
    public  function checkPluginAccess($usr_id=null): **string**

Reply:
did not listen to uncle bob ;-). Null as default probably wrong. Here especially. Always pass $usr_id to prevent side effects.

Reply:
fixed